### PR TITLE
Feature/fix filled using last cell

### PR DESCRIPTION
**Pull request by andykkt**
Fixed scrolling problem when keyboard is presenting with filledUsingLastCell attributes

### CODE REVIEW

**Comment by andykkt** [comment]
Changed to develop branch

File: BFWControls/Modules/Table/Controller/StaticTableViewController.swift [hunk 1]
```diff
@@ -78,22 +77,8 @@ class StaticTableViewController: UITableViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        if intrinsicHeightCells {
-            tableView.estimatedRowHeight = 44.0
-        }
```
Comment (barefeettom) [inline BFWControls/Modules/Table/Controller/StaticTableViewController.swift]: Are you sure you want to remove the `if intrinsicHeightCells` wrapper around this? I expect we want to set `estimatedRowHeight` is `intrinsicHeightCells` is `true`.

File: BFWControls/Modules/Table/Controller/StaticTableViewController.swift [hunk 2]
```diff
@@ -114,9 +99,21 @@ class StaticTableViewController: UITableViewController {
     // MARK: - UITableViewDelegate
 
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        let height = intrinsicHeightCells
+        var height = intrinsicHeightCells
             ? UITableViewAutomaticDimension
             : super.tableView(tableView, heightForRowAt: superIndexPath(for: indexPath))
+        // Check filledUsingLastCell attribute.
+        if filledUsingLastCell {
+            // Is this last cell?
```
Comment (barefeettom) [inline BFWControls/Modules/Table/Controller/StaticTableViewController.swift]: I don't think we need this comment and the one above. The lines they describe are fairly obvious.

File: BFWControls/Modules/Table/Controller/StaticTableViewController.swift [hunk 3]
```diff
@@ -114,9 +99,21 @@ class StaticTableViewController: UITableViewController {
...
+            if indexPath == lastCellIndexPath {
```
Comment (barefeettom) [inline BFWControls/Modules/Table/Controller/StaticTableViewController.swift]: Can we combine the above two `if` conditions, to reduce a level of indent?

File: BFWControls/Modules/Table/Controller/StaticTableViewController.swift [hunk 4]
```diff
@@ -114,9 +99,21 @@ class StaticTableViewController: UITableViewController {
...
+                let previousRowFrame = tableView.rectForRow(at: IndexPath(row: indexPath.row-1, section: indexPath.section))
```
Comment (barefeettom) [inline BFWControls/Modules/Table/Controller/StaticTableViewController.swift]: Please insert a space of both sides of the `-` operator, like you have for the `+` below.

**Review by barefeettom — CHANGES_REQUESTED** [review]

File: BFWControls/Modules/Table/Controller/StaticTableViewController.swift (hunk 2 above)
Comment (andykkt) [reply BFWControls/Modules/Table/Controller/StaticTableViewController.swift]: Fixed in latest commit.

File: BFWControls/Modules/Table/Controller/StaticTableViewController.swift (hunk 3 above)
Comment (andykkt) [reply BFWControls/Modules/Table/Controller/StaticTableViewController.swift]: Fixed in latest commit.

File: BFWControls/Modules/Table/Controller/StaticTableViewController.swift (hunk 4 above)
Comment (andykkt) [reply BFWControls/Modules/Table/Controller/StaticTableViewController.swift]: Fixed in latest commit.

File: BFWControls/Modules/Table/Controller/StaticTableViewController.swift (hunk 1 above)
Comment (andykkt) [reply BFWControls/Modules/Table/Controller/StaticTableViewController.swift]: if it doesn’t have estimatedRowHeight set, it actually causing infinite loop from numberOfRowInSection which will call heightForRowAt again.
How about we set estimatedRowHeight when user set *intrinsicHeightCells* or *filledUsingLastCell*? changed in latest commit.

**Pull request merged**